### DISCUSSION TITLE
Should the equalizer work on OpenBSD?

### DISCUSSION THREAD

**Discussion by vext01** [strawberrymusicplayer/strawberry#845] [Q&A] [answered]
Hi,

I'm an OpenBSD developer porting strawberry.

I've noticed that the EQ doesn't seem to do anything. Is it expected to work on OpenBSD?

Thanks

**Comment by jonaski** [comment, marked as answer]
I don't see any reason why that would behave differently on OpenBSD.
Did you check the "Enable equalizer" checkbox before starting playback?
